bots/tests-scan

```diff
@@ -51,6 +51,9 @@ REDHAT_VERIFY = {
     'selenium/explorer': [ 'master' ],
 }
 
+# Label: should a PR trigger external tests
+TRIGGER_EXTERNAL = "trigger-external"
```

Comment:
Label should be more descriptive. Perhaps "test-other" or "test-broadly"

Reply:
Hmm, "test-external"?

Reply:
Sure.